1.split_data/scripts/nbconverted/split_data.py

```diff
@@ -17,64 +17,70 @@
 from split_utils import get_features_data
 
 
-# ### Load data and set holdout/test parameters
+# ## Specify datasets, save path
 
 # In[2]:
 
 
-# load x (features) and y (labels) dataframes
-labeled_data_path = pathlib.Path("../0.download_data/data/labeled_data.csv.gz")
-labeled_data = get_features_data(labeled_data_path)
-print(labeled_data.shape)
+# datasets to split
+datasets = ["ic", "no_ic"]
 
-
-# In[3]:
+# make results dir for saving
+results_dir = pathlib.Path("indexes/")
+results_dir.mkdir(parents=True, exist_ok=True)
 
 
-# ratio of data to be used for testing (ex 0.15 = 15%)
-test_ratio = 0.15
+# ## Load, split, save data indexes
 
-# get indexes of training and testing data
-training_data, testing_data = train_test_split(
-    labeled_data,
-    test_size=test_ratio,
-    stratify=labeled_data[["Mitocheck_Phenotypic_Class"]],
-    random_state=1,
-)
-train_indexes = training_data.index.to_numpy()
-test_indexes = testing_data.index.to_numpy()
+# In[3]:
 
-print(f"Training data has shape: {training_data.shape}")
-print(f"Testing data has shape: {testing_data.shape}")
 
+for dataset in datasets:
+    print(f"Splitting data for {dataset} dataset")
+    
+    # load x (features) and y (labels) dataframes
+    labeled_data_path = pathlib.Path(f"../0.download_data/data/labeled_data__{dataset}.csv.gz")
+    labeled_data = get_features_data(labeled_data_path)
+    print(f"Dataset shape: {labeled_data.shape}")
+    
+    # ratio of data to be used for testing (ex 0.15 = 15%)
+    test_ratio = 0.15
```

Comment:
constants should be defined sooner in the notebook (and also not defined within a loop!)

Reply:
Good catch, fixed!